internal/operators/mce/mce_operator_test.go

```diff
@@ -78,19 +78,16 @@ var _ = Describe("MCE Operator", func() {
 		)
 	})
 	Context("ValidateCluster", func() {
-		mceMinOpenshiftVersion, err := getMinMceOpenshiftVersion(operator.config.OcpMceVersionMap)
-		Expect(err).ToNot((HaveOccurred()))
-
 		table.DescribeTable("validate cluster when ", func(cluster *common.Cluster, expectedResult api.ValidationResult) {
 			res, _ := operator.ValidateCluster(ctx, cluster)
 			Expect(res).Should(Equal(expectedResult))
 		},
 			table.Entry("Openshift version less than minimal",
```

Comment:
can we add a test case for pre-release version ? i.e 4.15-rc0

I've seen it fail in other operator - let make sure this is always working